libtsuba/src/ParquetWriter.cpp

```diff
@@ -205,3 +256,40 @@ tsuba::ParquetWriter::StoreParquet(
   desc->AddOp(std::move(future), uri.string());
   return katana::ResultSuccess();
 }
+
+katana::Result<void>
+tsuba::ParquetWriter::StoreParquet(
+    const katana::Uri& uri, tsuba::WriteGroup* desc) {
+  if (!opts_.write_blocked) {
+    KATANA_LOG_ASSERT(tables_.size() == 1);
+    return StoreParquet(tables_[0], uri, desc);
+  }
+
+  std::unique_ptr<tsuba::WriteGroup> our_desc;
+  if (!desc) {
+    auto desc_res = WriteGroup::Make();
+    if (!desc_res) {
+      return desc_res.error();
+    }
+    our_desc = std::move(desc_res.value());
+    desc = our_desc.get();
+  }
+
+  katana::Result<void> ret = katana::ResultSuccess();
+  for (uint64_t i = 0, num_tables = tables_.size(); i < num_tables; ++i) {
+    ret = StoreParquet(tables_[i], uri + fmt::format(".{}", i), desc);
```

Comment:
do these names/urls need to be sortable in a way that matches the original table order?

Reply:
They don't for the usecase driving this but that's a good idea, I'll add some leading zeros.